From: Robert Church <robert.church@windriver.com>
Date: Sun, 31 Dec 2023 12:49:27 -0600
Subject: Fix chart label for rbd-storage-init Job

Update the chart label on the rbd-storage-init Job to align with all the
other chart labels used throughout the chart.

This one-off implementation resulted in an invalid
label (ceph-csi-rbd-3.9.0+STX.14) based on the new chart auto-versioning
scheme. Using the common include mechanism provides a valid
label (ceph-csi-rbd-3.9.0_STX.14).

Signed-off-by: Robert Church <robert.church@windriver.com>
---
 charts/ceph-csi-rbd/templates/storage-init.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/charts/ceph-csi-rbd/templates/storage-init.yaml b/charts/ceph-csi-rbd/templates/storage-init.yaml
index 6abd8ec..0b36c58 100644
--- a/charts/ceph-csi-rbd/templates/storage-init.yaml
+++ b/charts/ceph-csi-rbd/templates/storage-init.yaml
@@ -344,7 +344,7 @@ spec:
       labels:
         heritage: {{ .Release.Service | quote }}
         release: {{ .Release.Name | quote }}
-        chart: "{{ .Chart.Name }}-{{- .Chart.Version }}"
+        chart: {{ include "ceph-csi-rbd.chart" . }}
     spec:
       serviceAccountName: {{ include "ceph-csi-rbd.serviceAccountName.provisioner" . }}
       restartPolicy: OnFailure
